linera-service/src/benchmark.rs

```diff
@@ -212,6 +212,9 @@ async fn benchmark_with_fungible(
         |((_, context, node_service), expected_balances)| {
             try_join_all(apps.iter().zip(expected_balances).map(
                 |((_, sender_context, _), expected_balance)| async move {
+                    if expected_balance == Amount::ZERO {
+                        return Ok(()); // No transfers: The app won't be registered on this chain.
+                    }
```

Comment:
Does this belong to this PR?

Reply:
Not really, I'll move it to a new one.

Reply:
https://github.com/linera-io/linera-protocol/pull/1450